docs/index.rst

```diff
@@ -202,3 +202,4 @@ Next Steps
 
    developer-guide/actions/index
    developer-guide/platform/index
+   developer-guide/sinks-guide
```

Comment:
I think this belongs under the "Robusta Engine Development" heading

![Screen Shot 2022-08-28 at 0 11 32](https://user-images.githubusercontent.com/494087/187048125-192b8cf1-f20d-4c38-8cdb-e7d9203397a4.png)